os/arch/arm/src/s5j/Kconfig

```diff
@@ -358,3 +358,33 @@ config S5J_SENSOR_PPD42NS
 	default n
 
 endmenu
+
+comment "IDLE PM Options"
+config S5J_IDLE_PM
+	bool "S5J IDLE PM"
+	default n
+	select ARCH_HAVE_IDLE_PM
+	---help---
+		Enable s5j idle pm support.
+
+config ARCH_IDLE_PM_NSTATE
+	int
+	default 3
+	depends on ARCH_HAVE_IDLE_PM
+	---help---
+		Number of sleep states provided by the SoC Power module
+
+config ARCH_IDLE_PM_STATE_COUNT_THRESH
+	string "IDLE PM STATE count threshold"
+	default "30,50,"
```

Comment:
If default states are 3, there should be clear description regarding why only 2 entries are accepted.

Reply:
have mentioned in Kconfig explanation